Resources/doc/configuration.md

```diff
@@ -21,3 +21,7 @@ sulu_redirect:
             model: Sulu\Bundle\RedirectBundle\Entity\RedirectRoute
             repository: Sulu\Bundle\RedirectBundle\Entity\RedirectRouteRepository
 ```
+
+Make sure that the user roles for redirects are set in Sulu for relevant users roles, otherwise the item is not visible in the navigation and the usrs are not able manage the redirects. 
```

Comment:
Thanks a lot for spending your time on this! I think it would be even better to add this section to the end of the `installation.md`. Can you move it there?

Reply:
done :)